src/TodoList.js

```diff
@@ -17,12 +17,13 @@ const todoList = [
     }
   ];
   
-const TodoList = () => {
+const TodoList = ({todoList}) => {
+  const {id, title} = todoList;
     return (
         <ul>
             {todoList.map(function(item) {
                 return (
```

Comment:
We need to move the id destructuring from the overall component to inside of this `map` callback: 

```suggestion
                const { id } = item;
                
                return (
```

Reply:
done

Reply:
done